src/main/ipc/react-devtools.ts

```diff
@@ -0,0 +1,30 @@
+import AdmZip from "adm-zip";
+import { ipcMain } from "electron";
+import fetch from "node-fetch";
+import { join } from "path";
+import { CONFIG_PATHS } from "src/util.mjs";
+import { RepluggedIpcChannels } from "../../types";
+
+const OUTPUT_PATH = join(CONFIG_PATHS["react-devtools"]);
+
+// https://github.com/facebook/react/issues/25843#issuecomment-1406766561
+const EXTENSION_URL =
+  "https://github.com/mondaychen/react/raw/017f120369d80a21c0e122106bd7ca1faa48b8ee/packages/react-devtools-extensions/ReactDevTools.zip";
+
+ipcMain.handle(RepluggedIpcChannels.DOWNLOAD_REACT_DEVTOOLS, async () => {
+  const arrayBuffer = await fetch(EXTENSION_URL)
```

Comment:
Can we check if it's already there before downloading it again? Also one we should consider (which ig could be done later) is figure out how to handle updates to RDT. Maybe a setting key for the version and if that's older than whatever version is in a certain const then we re-download when Discord is launched?

Reply:
> Can we check if it's already there before downloading it again?
What if the extension breaks, I don't know, the user deletes some file or changes something making it not loading. A redownload would overwrite any files, solving this possible issue. How would I check if the extension is downloaded or not? Should I save the zip individually, and check if it's there?

> Also one we should consider (which ig could be done later) is figure out how to handle updates to RDT. Maybe a setting key for the version and if that's older than whatever version is in a certain const then we re-download when Discord is launched?
Certainly later, if we will consider forking the modified RDT.